src/transformers/models/speecht5/modeling_speecht5.py

```diff
@@ -2414,7 +2542,8 @@ def _generate_speech(
     minlenratio: float = 0.0,
     maxlenratio: float = 20.0,
     vocoder: Optional[nn.Module] = None,
-) -> torch.FloatTensor:
+    output_cross_attentions: Optional[bool] = None,
```

Comment:
What does it mean to have a value of `None` for this param? Often for e.g. `output_attentions` it's used to take the default config values. As far as I can tell, it's only ever used as a `bool`

Reply:
Fixed